pages/dkp/kommander/2.1/install/air-gapped/index.md

```diff
@@ -160,14 +154,6 @@ Based on the network latency between the environment of script execution and the
 
 ## Install on Konvoy
 
-1. Kommander v2.1 installs with a dedicated CLI.
-
-1. Create an installation configuration file:
-
-    ```bash
-    kommander install --init > install.yaml
-    ```
-
 1. Adapt the configuration file for the air-gapped deployment by changing the `.apps.kommander` section. Ensure you use the actual version number everywhere `${VERSION}` appears:
```

Comment:
Because this is added to the pre-reqs up above, but this is some lines after that in the topic, I think we should just add a note here:

```suggestion
1. Adapt the configuration file created from running `kommander install --init > install.yaml` for the air-gapped deployment by changing the `.apps.kommander` section. Ensure you use the actual version number everywhere `${VERSION}` appears:
```